src/aleph_vrf/models.py

```diff
@@ -168,6 +166,20 @@ def from_vrf_response(
             message_hash=message_hash,
         )
 
+    @classmethod
+    def from_vrf_post_message(cls, post_message: PostMessage) -> "PublishedVRFResponse":
```

Comment:
Adding `from __future__ import annotations` as the first import removes the restriction to add quotes around type hints.
```suggestion
    def from_vrf_post_message(cls, post_message: PostMessage) -> PublishedVRFResponse:
```